deploy/ods-pipeline/charts/ods-pipeline-tasks/templates/task-ods-build-go.yaml

```diff
@@ -81,10 +81,14 @@ spec:
       description: Whether to skip SonarQube analysis or not.
       type: string
       default: "false"
+  {{- with ((.Values.go).sidecars) }}
```

Comment:
Not sure if `go` is too short here. Maybe nest under `tasks`? Or something like `buildGoTask`?

Reply:
I think I would stick with it for the moment: The functionality is actually configured through the top-level/umbrella chart `values.yaml` which has a surrounding `tasks` key in which the language/builder specifics are defined (c.f. https://github.com/opendevstack/ods-pipeline/blob/251ee7dec9cb6dba490609b0671f235167e8d361/deploy/ods-pipeline/values.yaml#L178-L222). What do you think?